test/integration/targets/prepare_vmware_tests/tasks/main.yml

```
@@ -0,0 +1,104 @@
+---
+
+- when: vcsim is defined
+  block:
+  - name: start vcsim
+    uri:
+      url: http://{{ vcsim }}:5000/spawn?cluster=2
```

```suggestion
      url: http://{{ vcsim }}:5000/spawn?cluster=2&ds=2
```
Adding 2 datastore might solve `vmware_datastore_facts` testcase.

---

I would prefer to keep the default resources as low as possible, for instance, we should not even have a `cluster=2` here. Instead, the resources should only be created on the fly when a test will consume them. This way we will avoid side-effect and speed up the run on regular lab.

This being said, I'm not sure if this can be applied to all the test cases.